pkg/minikube/constants/constants.go

```diff
@@ -184,10 +184,6 @@ var (
 
 	// ISOCacheDir is the path to the virtual machine image cache directory
 	ISOCacheDir = localpath.MakeMiniPath("cache", "iso")
```

Comment:
Should we do the same here ?

Reply:
Ah, I was under the impression we'd have two different iso names, but this is a simple enough change that we should do it.

Reply:
I guess either works ?

Reply:
Might _need_ to have different names, if they are going to be stored in the same place - like GitHub

Reply:
even if they have different names, having the a uniform cache directory system is probably the move anyway. i've added it.